mne/io/tests/test_read_raw.py

```diff
@@ -66,3 +70,22 @@ def test_read_raw_supported(fname):
     read_raw(fname, verbose=False)
     raw = read_raw(fname, preload=True)
     assert "data loaded" in str(raw)
+
+
+def test_split_name_ext():
+    """Test file name extension splitting."""
+    # test supported extensions
+    for ext in supported.keys():
+        assert split_name_ext(f"test{ext}")[1] == ext
+
+    # test unsupported extensions
+    for ext in ("this.is.not.supported", "a.b.c.d.e", "fif.gz.xyz"):
+        assert split_name_ext(f"test{ext}")[1] is None
```

Comment:
```suggestion
        assert split_name_ext(f"test{ext}")[1] is None
    for ext in readers:
        assert split_name_ext(f"test{ext}")[1] == ext
```

Reply:
Ahh yes or just change the above `supported` to `readers` might be even better, just didn't notice it